src/main/java/sx/blah/discord/modules/ModuleLoader.java

```diff
@@ -236,7 +242,7 @@ public static synchronized void loadExternalModules(File file) { // A bit hacky,
 							addModuleClass(classInstance);
 					}
 				}
-			} catch (NoSuchMethodException | IllegalAccessException | InvocationTargetException | IOException | ClassNotFoundException e) {
+			} catch (NoClassDefFoundError | NoSuchMethodException | IllegalAccessException | InvocationTargetException | IOException | ClassNotFoundException e) {
```

Comment:
No. NoClassDefinitionError is an error hence error in its name. Errors are not exceptions.